entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/impl/TimespanImpl.java

```diff
@@ -123,11 +123,6 @@ public void setDctermsHasPart(Map<String, List<String>> arg0) {
     public void setEnd(Map<String, List<String>> arg0) {
 	// TODO Auto-generated method stub
 	
-    }
-
-    @Override
-    public String getType() {
-        return "Timespan";
```

Comment:
please revert back this code, and use EntityTypes.Tmespan.getEntityType() instead of the hardcoded value

Reply:
Done.